docs/src/Flex.doc.js

```diff
@@ -57,7 +57,6 @@ card(
         description:
           'Establishes the main axis, thus defining the direction flex items are placed in the flex container.',
       },
-      { name: 'fit', type: 'boolean', defaultValue: false },
```

Comment:
Think this was accidentally listed in Flex

Reply:
From a previous version I think. Good catch!